src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -32,11 +44,18 @@ export interface IExpressionEnrichmentContainerProps {
         nameOfEnrichmentDirection?: string,
         count: number,
         color?: string
-    }[]
+    }[];
+    sampleKeyToSample: {
+        [uniqueSampleKey: string]: Sample;
+    }
+    groupSampleAxisData: IStringAxisData
     alteredVsUnalteredMode?: boolean;
-    store?: ResultsViewPageStore;
+    queriedHugoGeneSymbols?: string[];
+    oqlFilteredCaseAggregatedData?: { [uniqueSampleKey: string]: ExtendedAlteration[] };
 }
 
+const SVG_ID = "expression-enrichment-tab-plot-svg";
```

Comment:
I know this is how we did it but I think @alisman had said we shouldn't do that anymore, we should use a `ref` strategy to stay within react idioms

Reply:
you decided not to change it?